static/js/plotjs/smooth_spectra.js

```diff
@@ -12,8 +12,12 @@ const smoothing_func = (values, window_size) => {
     const idx_high = i + over < values.length ? i + over : values.length - 1;
     let N = 0;
     for (let j = idx_low; j <= idx_high; j += 1) {
-      N += 1;
-      output[i] += values[j];
+      // check value is not NaN
+      /* eslint no-self-compare: "off" */
+      if (values[j] === values[j]) {
```

Comment:
use isNaN? https://stackoverflow.com/questions/2652319/how-do-you-check-that-a-number-is-nan-in-javascript

Should also add a test which includes NaN in the input spectrum to see if you get the behavior you're looking for.

Reply:
See also https://github.com/profjsb/skyportal/blob/4b81f1537b526d45d935a67aae49af05d7552b1f/static/js/components/Periodogram.jsx#L106

Reply:
Check the behavior is tricky because the output is a bokeh plot.